### ISSUE TITLE
ServiceControl 5.2.x SagaAudit import fails to convert timespan

### ISSUE THREAD

**Issue by NChaganlal** [Particular/ServiceControl#4202]
#### Description

When the ServiceControl is importing the SagaAudit information a Timespan conversion error occurs and the message fails to import

#### Expected behavior
Saga audit information is imported successfully

#### Actual behavior
Saga audit information fails to import with a Timespan conversion error

#### Versions
NServiceBus 8.2.0
ServiceControl 5.2.0

Please list the version of the relevant packages or applications in which the bug exists.        
https://github.com/Particular/NServiceBus.SagaAudit/releases/tag/4.0.0

### Steps to reproduce

1. Create an endpoint with NServiceBus v8.2 with SagaAudit plugin v4.0.0 installed
2. Create a Saga on the endpoint
3. Start the Saga
4. Request a timeout of 2 days or a delay delivery message - this will generate a message with a payload that contains a timespan field with the "g" format like this `2:0:00:00`

Sample message
```c#
string json = """
              {
                  "ResultingMessages": [
                      {
                          "DeliveryAt": null,
                          "DeliveryDelay": "2:0:00:00"
                      }
                  ]
              }
              """;
```

6. The state of the saga will be recorded and trigger an audit message to be imported
7. The SagaAudit will fail because it can't parse the timestamp

### Relevant log output

```shell
Exception:

The input string was not in a correct Timespan format

StackTrace:

   at ServiceControl.SagaAudit.CustomTimeSpanConverter.ThrowFormatException() in /_/src/ServiceControl.SagaAudit/CustomTimeSpanConverter.cs:line 85
   at ServiceControl.SagaAudit.CustomTimeSpanConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options) in /_/src/ServiceControl.SagaAudit/CustomTimeSpanConverter.cs:line 63
   at System.Text.Json.Serialization.Metadata.JsonPropertyInfo`1.ReadJsonAndSetMember(Object obj, ReadStack& state, Utf8JsonReader& reader)
   at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.Metadata.JsonPropertyInfo`1.ReadJsonAndSetMember(Object obj, ReadStack& state, Utf8JsonReader& reader)
   at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.Serialization.Metadata.JsonTypeInfo`1.Deserialize(Stream utf8Json)
   at ServiceControl.Audit.Auditing.AuditPersister.ProcessSagaAuditMessage(MessageContext context) in /_/src/ServiceControl.Audit/Auditing/AuditPersister.cs:line 219
--- End of stack trace from previous location ---
   at ServiceControl.Audit.Auditing.AuditIngestion.OnMessage(MessageContext messageContext, CancellationToken cancellationToken) in /_/src/ServiceControl.Audit/Auditing/AuditIngestion.cs:line 197
   at NServiceBus.Transport.AzureServiceBus.MessagePump.ProcessMessage(ServiceBusReceivedMessage message, ProcessMessageEventArgs processMessageEventArgs, String messageId, Dictionary`2 headers, BinaryData body, CancellationToken messageProcessingCancellationToken) in /_/src/Transport/Receiving/MessagePump.cs:line 285

Source:

ServiceControl.Audit

TargetSite:

Void ThrowFormatException()
```

#### Workarounds

1. Update NServiceBus to v9 and keep ServiceControl on version 5.2.0
2. Keep NServiceBus v8.2 and rollback ServiceControl to version 5.1.0

**Comment by danielmarbach** [comment]
We've prioritized your issue near the top of our list and will work on it as soon as we can

**Comment by danielmarbach** [comment]
Will be released as part of 5.2.2
